Sources/StytchUI/StytchUIClient/ViewControllers/AuthHomeViewController/AuthHomeViewModel.swift

```diff
@@ -54,7 +54,10 @@ extension AuthHomeViewModel: AuthHomeViewModelProtocol {
         try await EventsClient.logEvent(
             parameters: .init(
                 eventName: "render_login_screen",
```

Comment:
```suggestion
                eventName: "render_consumer_login_screen",
```
Makes it a bit long, but let's the human know positively that this is Consumer even if they don't know the B2B one is `render_b2b_login_screen`.